account/account.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package account
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/network"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/key"
+)
+
+type Account struct {
+	*key.SoftKey
+}
+
+func NewAccount() (Account, error) {
+	k, err := key.NewSoft()
+	if err != nil {
+		return Account{}, err
+	}
+	return Account{
+		SoftKey: k,
+	}, nil
+}
+
+func Import(keyPath string) (Account, error) {
+	k, err := key.LoadSoft(keyPath)
+	if err != nil {
+		return Account{}, err
+	}
+	return Account{
+		SoftKey: k,
+	}, nil
+}
+
+func (a *Account) GetPChainAddress(network network.Network) (string, error) {
+	if a.SoftKey == nil {
+		return "", fmt.Errorf("SoftKey not initialized")
+	}
+	pchainAddrs, err := a.SoftKey.GetNetworkChainAddress(network, "P")
+	return pchainAddrs[0], err
+}
```

Comment:
I think there would be a lot of value of putting unit tests in this PR, especially to help illustrate usage.

Reply:
+1. Would be also easier to modify code later on without breaking core functionality

Reply:
Unit Tests can be added in subsequent PRs imo, especially as we are restructuring account and wallet in subsequent PRs: 
https://github.com/ava-labs/avalanche-tooling-sdk-go/pull/165/files#diff-fd8409dbfc4e5669f0b8ddb356e050f54d9a30100c2c397a24240b82ad81b351

https://github.com/ava-labs/avalanche-tooling-sdk-go/issues/174